src/GeoStatsBase.jl

```diff
@@ -80,6 +80,7 @@ include("errors.jl")
 include("statistics.jl")
 include("plotrecipes.jl")
 include("utils.jl")
+include("histograms.jl")
```

Comment:
please move this include to afte the `statistics.jl` include.

Reply:
Sure